hudi-common/src/test/java/org/apache/hudi/common/util/TestFSUtils.java

```diff
@@ -261,4 +289,87 @@ public void testLogFilesComparison() {
   public static String makeOldLogFileName(String fileId, String logFileExtension, String baseCommitTime, int version) {
     return "." + String.format("%s_%s%s.%d", fileId, baseCommitTime, logFileExtension, version);
   }
+
+  private void cleanupFiles(File[] cleanupFiles) {
```

Comment:
Once you use the basePath from the DFS started by the TestHarness and create files underneath, when the test dies, it will automatically clean those files. You shouldn't need to clean it here